packages/cli/src/util/deploy/create-deploy.ts

```diff
@@ -84,7 +89,7 @@ export default async function createDeploy(
     if (error.code === 'cert_missing') {
       const result = await generateCertForDeploy(
         output,
-        now,
+        client!,
```

Comment:
This might be a bug. What about if `client` is undefined?

Reply:
I don't have idea, this error is generated when the file transform to TypeScript 🤔

Reply:
So what about `if (!client) throw new Error ('Expected client but found undefined')`

Reply:
I agree, this solution It seems to work @styfle. I think that approach is the most appropriate for checking In If case `client` is that to `undefined `  👍